README.md

```diff
@@ -10,8 +10,7 @@ to interact with [GitHub’s REST API](https://developer.github.com/v3/) and
 [GitHub’s GraphQL API](https://developer.github.com/v4/guides/forming-calls/#the-graphql-endpoint).
 
 It uses [`@octokit/endpoint`](https://github.com/octokit/endpoint.js) to parse
-the passed options and sends the request using [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
-([node-fetch](https://github.com/bitinn/node-fetch) when the runtime has no native `fetch` API).
+the passed options and sends the request using [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).
```

Comment:
Shall we mention in the intro that a custom `fetch` function can be passed? Some thing like this

```suggestion
the passed options and sends the request using [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). You can pass a custom `fetch` function using the `request.fetch` option, see below.
```